.idea/misc.xml

```diff
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="EntryPointsManager">
+    <list size="3">
+      <item index="0" class="java.lang.String" itemvalue="org.springframework.stereotype.Component" />
+      <item index="1" class="java.lang.String" itemvalue="org.springframework.stereotype.Controller" />
+      <item index="2" class="java.lang.String" itemvalue="org.springframework.web.bind.annotation.GetMapping" />
+    </list>
+    <writeAnnotations>
+      <writeAnnotation name="org.springframework.beans.factory.annotation.Autowired" />
+    </writeAnnotations>
+  </component>
```

Comment:
This looks like a file that is generated by IntelliJ, do we need this?

Reply:
I agree, but don't know, as it wasn't in the .gitignore exclusions. It's easier to delete and ignore than it is to recreate if necessary.